HPL2/core/include/system/Platform.h

```diff
@@ -33,7 +33,11 @@ namespace hpl {
 
 	//-----------------------------------------
 
-    struct VideoComp : public std::binary_function<cVideoMode, cVideoMode, bool> {
+    struct VideoComp
+    #if __cplusplus < 201103L
+        : public std::binary_function<cVideoMode, cVideoMode, bool>
+    #endif
```

Comment:
do we need this I think I would just opt to drop this from the code base. I don't think I intend to compile this with c++ 2011.

Reply:
`std::binary_function` is for *pre* C++11, as in, it has been deprecated with C++11 (note the check does `<`, not `<=`).
I will remove it completely then.